tests/api_test.py

```diff
@@ -7309,8 +7309,8 @@ def foo(x, y):
     primals = (2., 3)
     tangents = (np.ones(()), np.zeros((), float0),)
     expected_tangents = (2., np.zeros((), float0))
-    self.assertArraysEqual(api.jvp(foo, primals, tangents),
-                           (primals, expected_tangents))
+    self.assertAllClose(api.jvp(foo, primals, tangents),
```

Comment:
Should this be assertArraysAllClose?

Reply:
No, `assertArraysAllClose` will have the same problem as `assertArraysEqual`. The key difference with `assertAllClose` is that it effectively does `pytree.flatten` and then calls `assertArraysAllClose` on each leaf.

Reply:
I see. LGTM